experimental/puppeteer-firefox/.ci/node10/Dockerfile.windows

```diff
@@ -0,0 +1,11 @@
+FROM microsoft/windowsservercore:latest
+
+ENV NODE_VERSION 10.18.1
+
+RUN setx /m PATH "%PATH%;C:\nodejs"
+
+RUN powershell -Command \
+    netsh interface ipv4 set subinterface 18 mtu=1460 store=persistent ; \
+    Invoke-WebRequest $('https://nodejs.org/dist/v{0}/node-v{0}-win-x64.zip' -f $env:NODE_VERSION) -OutFile 'node.zip' -UseBasicParsing ; \
+    Expand-Archive node.zip -DestinationPath C:\ ; \
+    Rename-Item -Path $('C:\node-v{0}-win-x64' -f $env:NODE_VERSION) -NewName 'C:\nodejs'
```

Comment:
While we're here, let's also remove the `experimental/puppeteer-firefox/.ci/node{6,8}` folders.

Reply:
They are removed now.